src/reducers/entities/admin.js

```diff
@@ -124,22 +124,23 @@ function samlCertStatus(state = {}, action) {
     }
 }
 
-function convertAnalyticsRowsToStats(data, name) {
+export function convertAnalyticsRowsToStats(data, name) {
     const stats = {};
+    const clonedData = [...data];
 
     if (name === 'post_counts_day') {
-        data.reverse();
-        stats[Stats.POST_PER_DAY] = data;
+        clonedData.reverse();
```

Comment:
Nit: As this has hardly has any performance overhead.

Looks like the mutation is caused by just by reverse. Could we just instead do `[...data].reverse()` so we don't clone if we are not reversing the order?